terraform/workspace_variables/preprod.tfvars.json

```diff
@@ -10,7 +10,7 @@
       "website_name": "find-a-lost-trn-preprod",
       "website_url": "https://find-a-lost-trn-preprod.london.cloudapps.digital/health",
       "test_type": "HTTP",
-      "contact_group": ["twd_tra_dqt"],
+      "contact_group": [249142],
```

Comment:
What's this change?

Reply:
I have separated the commit. Contact group ID was used to resolve a bug on status cake deployment.